### PR TITLE
Add Event Page

### PR DESCRIPTION
**Pull request by piyush-pawar-17**

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]
Looks good, not sure if the back button is to be at the top of the page though 🤔

**Comment by piyush-pawar-17** [comment]
Suggest changes.

**Comment by anshumanv** [comment]
Can't find an apt place for now, let it be.

**Comment by piyush-pawar-17** [comment]
OK

File: src/sass/components/_button.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.btn {
+	background-color: $background-color;
+	color: $color-primary;
+	padding: 0.5em 2em;
+	border: 1px solid $color-primary;
```
Comment (aashutoshrathi) [inline src/sass/components/_button.scss:5]: Make it 0.15rem instead.

File: src/sass/components/_button.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	border-radius: 3px;
```
Comment (aashutoshrathi) [inline src/sass/components/_button.scss:6]: Make this `0.3rem`

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
- Font LGTM :rocket: 
- I think we should make keys bold; keys = {'Prize', 'Venue'....}
- Shift whole thing to bit more right or increase padding around, so that the social icons don't look ugly on small screens.
- For Back Button (.btn queries): I have left few comments.

Are you using tabs (sized-8) for CSS files?

File: src/sass/components/_button.scss (hunk 1 above)
Comment (piyush-pawar-17) [reply src/sass/components/_button.scss:5]: Using `rem` for such small values don't make sense. It won't affect anything.

File: src/sass/components/_button.scss (hunk 2 above)
Comment (piyush-pawar-17) [reply src/sass/components/_button.scss:6]: Same applies for this. If the value is less than `10px`, it's ok to use `px` rather than `rem`.

**Comment by piyush-pawar-17** [comment]
- @aashutoshrathi making the keys bold would disturb the visual flow and it may be painful to read. I suggest to keep it as it is.
- I use 4 tabs.

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**